src/docs/commands_keybindings.md

```diff
@@ -1,122 +1,138 @@
 ---
-title: Commands and Keybindings
+title: Commands/Menus/Keybindings
 ---
 
-# Commands and Keybindings
+# Commands, Menus and Keybindings
 
-Theia can be extended in a number of different ways. Commands allow packages to contribute unique commands that can then be called by other packages. It's also possible to add keybindings and contexts to these commands, so that they can only be called under certain conditions (window focus, current selection etc.).
+Commands are runnable actions defined by an ID, a name and the function to be executed. Commands can be triggered via the quick access bar, they can be bound to keybindings or menu items and they can be called programmatically. Commands can be context sensitive, so that they can only be called under certain conditions (window focus, current selection etc.).
```

Comment:
> Commands can be context sensitive

Elaborate on how? Commands can be controlled primarily during registration with `isVisible` and `isEnabled`. We can also mention `isToggled` to be thorough.

Reply:
good catch, I added this with the last commit